manim/mobject/three_dimensions.py

```diff
@@ -153,6 +156,81 @@ def set_fill_by_checkerboard(self, *colors, opacity=None):
             face.set_fill(colors[c_index], opacity=opacity)
         return self
 
+    def set_fill_by_value(self, axes: "Mobject", colors: Union[Iterable[Color], Color]):
+        """Sets the color of each face of a parametric surface to a color relative to it's z-value
+
+        Parameters
+        ----------
+        axes :
+            The axes for the parametric surface.
```

Comment:
I think this needs some more explanation, like, to explain how the axe will be used (basically to retrieve the z value and to map them into a color).

Reply:
I agree and will update via a commit.